int32 state
geometry_msgs/Point position
